format/hint/schema.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "value": {
+      "type": ["string","null"]
```

Comment:
I'm not sure about this one.
I've done that because if you export the Exercise without the solutions, the `value / text` property is not exported.

I will make the propery optional and will manually validate it where needed (e.g. save hint).